index.bs

```diff
@@ -3882,6 +3882,8 @@ It mirrors some fields of the {{PublicKeyCredential}} object returned by
 
         This mirrors the {{Credential/type}} field of {{PublicKeyCredential}}.
 
+        Note: If all {{PublicKeyCredentialDescriptor}} elements in {{PublicKeyCredentialRequestOptions/allowCredentials}} are ignored then that MUST result in an error since an empty {{PublicKeyCredentialRequestOptions/allowCredentials}} is semantically distinct.
```

Comment:
I know this is already merged, but what error should be thrown? Following the algorithm steps, this would result in a timeout after all available authenticators get skipped. This is not ideal if you're trying to figure out that your webauthn call is not working since it would show up as a generic `NotAllowedError` tho 🤔